frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -287,7 +362,3 @@ export class CatalogTileViewPage extends React.Component {
     );
   }
 }
-CatalogTileViewPage.displayName = 'CatalogTileViewPage';
```

Comment:
nit:
This is a pattern used a lot in console and we shouldn't change it unless we have a better approach.

Reply:
I removed this when shifted from jsx->tsx

Reply:
Yes you removed it. Which is why I commented that we shouldn't be removing it unless you have a better solution.

Reply:
I haven't looked to see if it's needed, but if you want to know more about what it is doing @abhinandan13jan please read https://reactjs.org/docs/react-component.html#displayname

Reply:
I've added displayName as a static variable as it fails the type check in the current code.... @andrewballantyne @christianvogt need LGTM & approve